.github/workflows/deploy.workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: qiskit.org - build and deploy
+
+on:
+  # run when action is manually triggered
+  workflow_dispatch:
+  # run on pushes to master branch
+  push:
+    branches: [ master ]
+  # run daily at 08:00am UTC (on the default or base branch)
+  # https://crontab.guru/#0_8_*_*_*
+  schedule:
+    - cron: '0 8 * * *'
+
+jobs:
+  # job id/key
+  build-and-deploy:
+    # job name
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - run: npm ci
+      - run: npm run generate
+
+      - name: Deploy
+        run: cd deploy && ./deploy_cos.sh
+        shell: bash
+        env:
+          AIRTABLE_API_KEY: ${{ secrets.AIRTABLE_API_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          IBM_CLOUD_API_KEY: ${{ secrets.IBM_CLOUD_API_KEY }}
+          IBM_CLOUD_ORGANIZATION: ${{ secrets.IBM_CLOUD_ORGANIZATION }}
+          IBM_CLOUD_SPACE: ${{ secrets.IBM_CLOUD_SPACE }}
+          QISKIT_SEGMENT_APP_KEY: ${{ secrets.QISKIT_SEGMENT_APP_KEY }}
+          QISKIT_SEGMENT_SCRIPT: ${{ secrets.QISKIT_SEGMENT_SCRIPT }}
```

Comment:
I think the codebase does not use these values. Could you double-check before merging? If we are not using them, let's remove them.

Reply:
removed unused variables